lib/ovirt_metrics.rb

```diff
@@ -59,7 +59,8 @@ def self.host_realtime(host_id, start_time = nil, end_time = nil)
     host_realtime_metrics_to_hashes(metrics, nic_metrics)
   end
 
-  private
+  # The methods below this line are PRIVATE not meant to be used out of the scope
```

Comment:
I always try to make a PR as small as possible and sometimes just ignore rubocop
so this change has nothing to do with the scope of this commit / PR
but up to you 😄 it's not super important